components/image-upload.tsx

```
@@ -136,15 +143,17 @@ export function ImageUpload({
         {uploading ? (
           <>
             <div className="h-10 w-10 mb-2 flex items-center justify-center">
-              <div className="animate-spin rounded-full h-8 w-8 border-b-2 border-gray-400"></div>
+              <div className="animate-spin rounded-full h-8 w-8 border-b-2 border-gray-400 aria-label='Uploading files...'"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)